Assets/MixedRealityToolkit.Staging/UnityAR/Editor/ConfigurationChecker/Microsoft.MixedReality.UnityAR.ConfigurationChecker.asmdef

```diff
@@ -1,6 +1,8 @@
 {
-    "name": "Microsoft.MixedReality.UnityAR.ConfigCheck.asmdef",
-    "references": [],
+    "name": "osoft.MixedReality.UnityAR.ConfigCheckheck",
```

Comment:
```suggestion
    "name": "Microsoft.MixedReality.UnityAR.ConfigCheck.asmdef",
```

Reply:
i we dont want the asmdef at the end.. i will fix the start of the file